tutorials/code_submitter.md

```diff
@@ -0,0 +1,27 @@
+---
+layout: page
+title: How to Submit Code
+---
+
+# How to Submit Code
+
+For virtual aspects of the competition, such as the [Virtual League][virtual-league], you will submit your code using the [code-submitter][code-submitter].
+Your code will then be run alongside other teams' code in the [simulator][simulator].
+
+To prepare your code for upload, you should create a zip archive containing all the code you want to submit.
```

Comment:
Do we want to give any information about how to make a zip archive? Even if that's something like "on windows, you can do this by selecting all the files you wish to include, right clicking, and selecting archive files"

Reply:
What level of competency do we assume? It might be good however I assume that we would then want to also include guides for Linux and MacOS?

Reply:
I thought about this, but opted not to for simplicity/speed ... and intended to leave a comment about it but then forgot. Very open to that being added, but felt this page was valuable enough without it and I'm not well placed right now to chase down exactly how to do this across platforms.

Reply:
We typically provide info for Windows & MacOS. I think omitting Linux is _probably_ ok (as Linux feels likely to only be personal devices and the plethora of desktop environments makes it harder anyway), though I do think we should start including Chromebooks (see https://github.com/srobo/docs/issues/655).

Reply:
Sounds good. More info can't hurt.

Reply:
For clarity: I am still proposing that we aim to ship this PR without any platform walkthroughs and that we can add that later if/when needed.

Reply:
I'm happy with that.

Reply:
There are instructions for this here: https://studentrobotics.org/docs/tutorials/getting_code_on_the_robot

Reply:
That isn't quite what we're after as we specifically are talking about instructions for creating a `.zip` folder